### ISSUE TITLE
FP_MinusInfty > 0 returns True

### ISSUE THREAD

**Issue by nibrunie** [kalray/metalibm#11]
Comparisons of core.special_values such as signed infinities return erroneous results
when compared with numerical values:
```
from metalibm_core.core.special_values import FP_MinusInfty
from metalibm_core.core.ml_formats import ML_Binary32

print(FP_MinusInfty(ML_Binary32) > 0)
```

**Comment by nibrunie** [comment]
Partially fixed in 986e37f0dd117892de4cdd8958b0a2826b65817e
Now comparison raise NotImplementedError

**Comment by nibrunie** [comment]
Fixed by commit 5e7efd6 integrated in master branch
unit test added in commit 56919f6 (also integrated in master)
